yarn.lock

```diff
@@ -14043,7 +14043,7 @@ react-native-web@0.12.3, react-native-web@^0.12.3:
     prop-types "^15.6.0"
     react-timer-mixin "^0.13.4"
 
-react-native-windows@^0.63.0:
+react-native-windows@^0.63.0, react-native-windows@^0.63.0-0:
```

Comment:
Was this added by yarn? This doesn't seem right, and AFAIK the `-0` suffix breaks the lookup of higher versions.

Reply:
-0 means to allow any prerelease build as well. It can still resolve to non-prerelease versions though.